simulators/portal-interop/src/main.rs

```diff
@@ -443,3 +455,33 @@ dyn_async! {
     }
 }
 
+dyn_async! {
+    async fn test_find_nodes_256_distance<'a>(test: &'a mut Test, client_a: Client, client_b: Client) {
+        let target_enr = match client_b.rpc.node_info().await {
+            Ok(node_info) => node_info.enr,
+            Err(err) => {
+                test.fatal(&format!("Error getting node info: {err:?}"));
+                return;
+            }
+        };
+
+        let (_, enr) = generate_random_remote_enr();
```

Comment:
What guarantees that the generated random ENR will always fall in the desired distance?

Reply:
I thought that 256 meant global radius. So it would just dump its table. I can look into it more and confirm my assumptions though.